slovak/marsmelaci.md

```diff
@@ -6,4 +6,6 @@ Jedno nevšedné ráno sa zobudíš v divnej miestnosti, ktorá má steny z mars
 
 [Pokúsiš sa zachrániť rybku](zachrana/zachrana-rybky.md)
 
+[Pokúsiš sa zachrániť seba](zachrana/zachrana-seba.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.